docs/database/schema-alternative.sql

```diff
@@ -0,0 +1,101 @@
+--
+-- Copyright (C) 2016 Red Hat, Inc.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--         http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+CREATE EXTENSION hstore;
+
+-- ==== Instances ====================================================================
+
+-- An action instance is the single entry point in this sub domain. It is referenced
+-- by an integration and has all configuration needed for creating an instance of
+-- a fully configured Camel connector
+CREATE TABLE action_instance
```

Comment:
This table is for me the combination of `configured_connector` configuration which is specific for a connector (mostly or exclusively defined by the expert dev) and the action configuration (defined by the citizen user) and the natural hand-over point to integrations. It contains all _configuration_ relevant information.

The alternative to keep the connector specific configuration in the DB and the action specific config in the integration YAML's is imo not favourable as you have to query now two different stores for getting a picture for the full configuration.

Also, I think that the configuration values in the YAMLs (not the structure) should come from the DB (as it is supposed to be changeable by a cit user, too, who has no idea of 'git'). So w.r.t parameter configuration, the DB is for me the authoritative source.